simpeg_drivers/constants.py

```diff
@@ -144,6 +136,7 @@
         "value": "",
         "optional": True,
         "enabled": True,
+        "tooltip": "Select a topography object to define the active cells for inversion.  The user may also provide active cells directly by disabling the parameter.  If no elevation channel is selected, the topography will be set from the geometry of the selected object.",
```

Comment:
Not sure a comment this long will render well. Just focus on the parameter itself and not its children.

Reply:
well, we could format the tooltip window better..  but sure - I'll shorten

Reply:
done everywhere